ts/Stock/StockTools/StockToolsDefaults.ts

```diff
@@ -1260,9 +1260,9 @@ const stockTools: StockToolsOptions = {
             }
         },
         /**
-         * Wether the stock tools toolbar is visible.
+         * Whether the stock tools toolbar is visible.
          *
-         * @type {boolean}
```

Comment:
Removing that `@type` was on purpose? 🤔

Reply:
Yes https://github.com/highcharts/highcharts/pull/21204#discussion_r1629683499